t/t2601-job-shell-standalone.t

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+#
+test_description='Test flux-shell in --standalone mode'
+
+. `dirname $0`/sharness.sh
+
+FLUX_SHELL=${FLUX_BUILD_DIR}/src/shell/flux-shell
+
+#  Set path to jq
+#
+jq=$(which jq 2>/dev/null)
+test -n "$jq" && test_set_prereq HAVE_JQ
+
+PMINFO=${FLUX_BUILD_DIR}/src/common/libpmi/test_pminfo
```

Comment:
nit: here and elsewhere, should this be "PMIINFO", with two I's in the middle?  I read this as "PM INFO" at first.